lab-cameron/index.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
I won't dock points for this since instructions weren't clear, but your ``greet`` functions should actually be imported and invoked in here as opposed to in your ``lib/greeting.js`` file. See me if you're unclear.